README.md

```diff
@@ -78,15 +78,20 @@ CriticalMAAS program.
 
 ### Geologic metadata curation
 
-- [`UW-Macrostrat/macrostrat-xdd`](https://github.com/UW-Macrostrat/macrostrat-xdd):
```

Comment:
Since this repo is just a readme, essentially, maybe we should remove the link and replicate the contents here?

_Or, this is where @bxia68 should place the integration code? It's certainly well-named!_

Reply:
Yes, we shouldn't link to an empty repo here. Where is the relevant code for the Ask-xDD interface etc? Can we link to that? cc @JasonLo

Reply:
I'm uncertain about the appropriate section for these 2 repositories. I'll defer to Devin's judgment on this matter.

- [`UW-xDD/text2graph_llm`](https://github.com/UW-xDD/text2graph_llm): An experimental tool designed to transform textual data into structured graph representations, leveraging the power of Large Language Models (LLMs) to identify and extract relationship triplets from text. It focuses on extracting relationships between locations and geological entities from the text.

- [`UW-Madison-DSI/ask-xDD`](https://github.com/UW-Madison-DSI/ask-xDD): Ask-XDD, developed in collaboration with the CHTC xDD team, offers an easy-to-use chat interface and API endpoint for accessing academic information via Retrieval-Augmented Generation (RAG). It helps bridge the gap between complex academic content and users, including laypeople and researchers. Our current prototype, available at http://cosmos0001.chtc.wisc.edu:8501/, currently covers topics such as geoscience, climate change, and COVID-19.